libavformat/rtpdec.c

```diff
@@ -640,6 +640,9 @@ static int rtp_parse_packet_internal(RTPDemuxContext *s, AVPacket *pkt,
         /* calculate the header extension length (stored as number
          * of 32-bit words) */
         extlen = (AV_RB16(buf + 2) + 1) << 2;
+        /* should also take the four-octet extension header into
+         * account */     
+        extlen += 4;    
```

Comment:
@tonykwok Hello. Thank you very much for your attention and your PR. I am not sure but hasn't `four-octet extension header` already been applied above?:
```c
len   -= 4 * csrc;
buf   += 4 * csrc;
```
or would you like to have it as a part of extension data?